scripts/sommelier

```diff
@@ -151,7 +151,17 @@ install_app() {
     EXE_NAME=$(basename "${LOCAL_INSTALL_EXE}")
     yes | yad --progress --title="Installing ${EXE_NAME}" --progress-text= --width=400 --center --no-buttons --auto-close --auto-kill --on-top --pulsate &
     INSTALL_YAD_PID="$!"
-    "${WINELOADER}" "${LOCAL_INSTALL_EXE}" "${ARR_INSTALL_FLAGS[@]}"
+    FILE_END="${EXE_NAME##*.}"
+    FILE_TYPE="${FILE_END,,}" # force lowercase
+    if [[ "$FILE_TYPE" == 'exe' ]]; then
+        "${WINELOADER}" "${LOCAL_INSTALL_EXE}" "${ARR_INSTALL_FLAGS[@]}"
+    elif [[ "$FILE_TYPE" == 'msi' ]]; then
+        "${MSIEXEC}" /i "${LOCAL_INSTALL_EXE}" "${ARR_INSTALL_FLAGS[@]}"
```

Comment:
I would prefer `"${WINELOADER}" msiexec /i ...` here. It's better to let wine figure out what the binary is for msiexec instead of assuming it's also in `/bin`.

Reply:
I never knew you could call msiexec from wine. I made the change and confirmed that it works.